changelog.d/20240222_174417_abdul.muqadim_binary_file_detection_update.md

```diff
@@ -0,0 +1 @@
+- [Improvement] This is a non-breaking change. Enhanced is_binary_file function in env file for better binary file detection. (by @Abdul-Muqadim-Arbisoft)
```

Comment:
This changelog entry is not very informative. As a Tutor user or plugin developer, what change(s) should I expect? Will some files now be considered as text/binary, that previously were not? If there is no change, what's the purpose of this PR?